src/controllers/queryRunner.ts

```diff
@@ -216,7 +230,9 @@ export default class QueryRunner {
             this._uriToQueryPromiseMap.delete(result.ownerUri);
         }
         this._statusView.executedQuery(result.ownerUri);
-        this.eventEmitter.emit('complete', Utils.parseNumAsTimeString(this._totalElapsedMilliseconds));
+        let hasError = false;
```

Comment:
nit: combine this line with the subsequent.  Why init to false and then immediately reassign a vale?